client/http/client.go

```diff
@@ -62,10 +62,13 @@ func (client *HttpClient) Post(path string, request interface{}, response interf
 }
 
 func (client *HttpClient) Put(path string, request interface{}, response interface{}) error {
-	result, err := client.request().
-		SetBody(request).
-		SetResult(response).
-		Put(path)
+	req := client.request().SetBody(request)
```

Comment:
While running the integration test the code crashed.
SetResult with nil is invalid. Added an "if" to resolve it.